src/services/DataService.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  AirImpurityDataClass,
+  type SensorData,
+  SensorDataClass
+} from '@/types/models/SensorData';
+
+const locationsURI = 'http://localhost:8080/sampleData';
+export default async function getRecent() {
+  const res = await fetch(locationsURI);
```

Comment:
Use the `axios` library instead of `fetch`. The `axios` library is already added to the project.